internal/utils/logging_test.go

```diff
@@ -83,6 +83,27 @@ func TestConnaisseurLogLevel(t *testing.T) {
 	}
 }
 
+func TestConnaisseurLogFormat(t *testing.T) {
+	var testCases = []struct {
+		logFormat string
+		want      constants.ConnaisseurLogFormat
+	}{
+		{
+			logFormat: "json",
+			want:      constants.LogFormatJson,
+		},
+		{
+			logFormat: "json-pretty",
+			want:      constants.LogFormatJsonPretty,
+		},
```

Comment:
```suggestion
		{
			logFormat: "json-pretty",
			want:      constants.LogFormatJsonPretty,
		},
		{
			logFormat: "not valid",
			want:      constants.LogFormatJsonPretty,
		},
```
maybe also default test?